### PR TITLE
Better way to control checkboxes using labels.

### PR DESCRIPTION
**Pull request by sibprogrammer**
Signed-off-by: Alexei Yuzhakov <sibprogrammer@gmail.com>

### CODE REVIEW

**Comment by FrenchBen** [comment]
I'm not sure what this actually adds?
It seems that you're adding a label to then over-write the label font and make it normal again?

diff
Top is your PR
Bottom is Kitematic 0.10.3

**Comment by sibprogrammer** [comment]
This brings an ability to click on checkbox label instead of trying to click on checkbox itself only. In the other words "Allocate a TTY for this container" also become clickable. It's a common pattern for apps, that not only the checkbox itself is clickable, but the whole label too.

**Comment by FrenchBen** [comment]
Any chance you'd consider doing the same thing for the Preference panel as well?

**Comment by sibprogrammer** [comment]
Ok, I'll check it later today :)

**Comment by sibprogrammer** [comment]
PR has been updated.

**Comment by FrenchBen** [comment]
LGTM

File: src/components/Preferences.react.js [hunk 1]
```diff
@@ -41,14 +41,14 @@ var Preferences = React.createClass({
       vmSettings = (
         <div>
           <div className="title">VM Settings</div>
-          <div className="option">
+          <label className="option">
```
Comment (FrenchBen) [inline src/components/Preferences.react.js]: Anyway to not use an inline element as a block-level element?

Comment (sibprogrammer) [inline src/components/Preferences.react.js]: Yes, it's possible using "label for=...", but it'll be necessary to assign the IDs for input. Is it ok for you?

Comment (FrenchBen) [inline src/components/Preferences.react.js]: Looks better IMHO - Thanks!

**Comment by FrenchBen** [comment]
@jeffdm PTAL

**Comment by sibprogrammer** [comment]
bump :)

**Comment by FrenchBen** [comment]
Thanks for the ping, going to merge it, as it LGTM

**Pull request merged**